.github/workflows/build-release-zip-file.yml

```diff
@@ -1,4 +1,4 @@
-name: Build release zip file
+name: 'Release: Build ZIP file'
```

Comment:
Small request for this change: prefix the workflows files names with `release-`.

Reply:
Wow. This is truly an early review 🤣. Thanks @kalessil!
I think it makes perfect sense. I'll push that and update this PR to mark it as ready for review.